fritzconnection/lib/fritzwlan.py

```diff
@@ -192,17 +235,22 @@ def beacontype(self):
         settings and `None, 11i, 11iandWPA3, OWETrans` for the guest
         network.
         """
-        return self.get_info()['NewBeaconType']
+        return self.get_info()["NewBeaconType"]
 
     @property
     def channel(self):
         """The WLAN channel in use"""
-        return self.channel_info()['NewChannel']
+        return self.channel_info()["NewChannel"]
 
     @property
     def alternative_channels(self):
-        """Alternative channels (as string)"""
-        return self.channel_info()['NewPossibleChannels']
+        """Alternative channels (as string)."""
+        return self.channel_info()["NewPossibleChannels"]
+
+    @property
+    def get_wifi_qr_code(self, kind):
+        """Returns a file-like object providing a bytestring representing a qr-code for WLAN access."""
+        return get_wifi_qr_code(self, kind)
```

Comment:
This will result in a recursion, even if the method is a property. Furthermore properties should be named like attributes and like attributes properties should not get arguments.